src/modules/navigator/navigator.h

```diff
@@ -162,7 +162,7 @@ class Navigator : public ModuleBase<Navigator>, public ModuleParams
 	void reset_vroi() { _vroi = {}; }
 
 	bool home_alt_valid() { return (_home_pos.timestamp > 0 && _home_pos.valid_alt); }
-	bool home_position_valid() { return (_home_pos.timestamp > 0 && _home_pos.valid_alt && _home_pos.valid_hpos); }
+	bool home_position_valid() { return (_home_pos.timestamp > 0 && _home_pos.valid_alt && _home_pos.valid_hpos && _home_pos.valid_lpos); }
```

Comment:
RTL uses home lat/lon after checking `home_position_valid()`.

https://github.com/PX4/PX4-Autopilot/blob/e8cefcafbcb93f83680505e71ca3bb2a290626c1/src/modules/navigator/rtl.cpp#L79-L96

Reply:
Yes, but that's fine because `_home_pos.valid_hpos` means that lat/lon are set. What's the issue?

Reply:
Sorry, misread the diff on my phone and thought you replaced hpos with lpos.

Good to go.